### PR TITLE
Bumped react film and set flippers to blur on click 

### LINKED ISSUE #1673: Suggested actions navigation arrows should indicate focus visually
After clicking on the horizontal navigation arrow in the quick answer area, if you don't click anything else and move the cursor away we get this behavior:

1. The CLICKED arrow doesn't disappear (still visible)
2. The other arrow disappears (becomes invisible)

For better understanding I am including this screenshot: 
screenshot at jan 31 10-17-19

And forgot to add that the change in styleSet doesn't affect these arrows

ANOTHER BUG:
Another bug (that is included in version: `"botframework-webchat": "^4.2.1-master.0dffd85",`) is that the actions have quotes around them (you can notice them on the picture above) , while I am receiving them without quotes from directLine (checked in the json that I received).

BTW: I know that 4.2.1 is not official yet, but 4.2.0 is too broken for my implementation, sry.

### PR DESCRIPTION
**Pull request by tdurnford**
Fixes #1673
## Changelog Entry
-  `component`: Bumps [`react-film@2.0.0`](https://npmjs.com/package/react-film/), by [@tdurnford](https://github.com/tdurnford) in PR [#26xx](https://github.com/microsoft/BotFramework-WebChat/pull/26xx)
-  Fixes [#1673](https://github.com/microsoft/BotFramework-WebChat/issues/1673). Configured suggested action and carousel flippers to blur on click, by [@tdunford](https://github.com/tdurnford) in PR [#26xx](https://github.com/microsoft/BotFramework-WebChat/pull/26xx)

## Description
Bumped react film to the most recent master release, and configured suggested action and carousel flippers to blur on click.

-  [ ] Testing Added

### CODE REVIEW

File: packages/component/src/Activity/CarouselLayout.js [hunk 1]
```diff
@@ -1,4 +1,8 @@
-import { Composer, Context as FilmContext, createBasicStyleSet, Flipper } from 'react-film';
+// import { Composer, Context as FilmContext, createBasicStyleSet, Flipper } from 'react-film';
```
Comment (tdurnford) [inline packages/component/src/Activity/CarouselLayout.js]: Remove

**Comment by compulim** [comment]
Looks like bumping to `react-film@2.0.2` fixed the problem.

But I am not able to kickoff a build based on your branch, could you take a look?

**Comment by corinagum** [comment]
https://media1.tenor.com/images/bb08b4da43e22c34a10a6df775bfd391/tenor.gif?itemid=12849217

**Comment by marq47** [comment]
Hi, there would be some way to customize in WebChat the carousel/suggested actions flipper behaviour changing react-film properties like autoHide (we need this property will be false), dir, showScrollBar, autoCenter, flipperBlurFocusOnClick, etc?

Now with styleSet we only can customize colors with Transcript overlay buttons properties (transcriptOverlayButtonBackground, transcriptOverlayButtonBackgroundOnFocus, transcriptOverlayButtonBackgroundOnHover, etc).

Thank you very much for your attention!!

**Comment by tdurnford** [comment]
@marq47 can you open a new feature request and add details on what you would like to see changed. This thread isn’t going to get much attention since the PR has already been merged.

**Pull request merged**